SparseGrids/tasgridUnitTests.cpp

```diff
@@ -45,6 +47,15 @@
 
 #ifdef Tasmanian_ENABLE_DPCPP
 #include <CL/sycl.hpp>
+#include <string>
```

Comment:
Lots of includes that are included transitively from `tsgEnumerates.hpp` and visible pretty much all throughout the code.

Also, be careful when adding code. This block sits inside `#ifdef Tasmanian_ENABLE_DPCPP` when it has nothing to do with DPC++.

Reply:
Removed all redundant includes in the latest commit.